python/hail/methods/misc.py

```diff
@@ -185,6 +185,61 @@ def rename_duplicates(dataset, name='unique_id'):
            intervals=oneof(Interval, listof(Interval)),
            keep=bool)
 def filter_intervals(ds, intervals, keep=True):
+    """Filter rows with an interval or list of intervals.
+
+    .. note::
+
+    Requires the dataset to have a partition key of type :class:`.TLocus`.
```

Comment:
This should be indented if part of a directive (example `.. note::`). However, this should be
`.. include:: ../require_tvariant.rst` . Not certain this is the correct path. Check other methods in the same package.

Reply:
FilterIntervals now only checks that the partition key is a locus, so the key doesn't have to be of form `[['locus'], 'alleles']` anymore.